packages/daemon/test/runner/helper.test.ts

```diff
@@ -120,4 +121,34 @@ describe('test the app service', () => {
     assert.ok(called, 'function call check');
     assert.ok(throwError, 'error call check');
   });
+  it('#should copy the folder successfully', async () => {
+    const src = join(__dirname, 'helper.test.ts');
+    const dest = join(__dirname, 'dest.ts');
+    const copyStub = sinon.stub(fs, 'copyFile');
+    const chmodStub = sinon.stub(fs, 'chmod');
+    try {
+      await assertNode.doesNotReject(async () => {
+        await helper.copyDir(src, dest);
+      }, 'copyDir should not be rejected');
+      assert.ok(copyStub.calledOnce, 'copyFile should be caled only once');
+      assert.ok(chmodStub.calledOnce, 'chmod should be caled only once');
+    } finally {
+      await fs.remove(dest);
```

Comment:
We should mock the fs operation to keep unit test not side effect.

Reply:
done